tests/jobs/scala/src/main/scala/S3Job.scala

```diff
@@ -1,22 +1,42 @@
+import scopt.OptionParser
 import org.apache.spark.SparkContext
 import org.apache.spark.SparkContext._
 import org.apache.spark.SparkConf
 
 object S3Job {
   def main(args: Array[String]): Unit = {
-    val conf = new SparkConf().setAppName("S3 Test")
-    val sc = new SparkContext(conf)
+    object config {
+      var readurl: String = null
+      var writeurl: String = null
+      var countonly: Boolean = false
+    }
 
-    val readURL = args(0)
-    val writeURL = args(1)
-    println(s"Reading from ${readURL}.  Writing to ${writeURL}.")
+    val parser = new OptionParser[Unit]("S3 Job") {
+      opt[String]("readUrl").action((x, _) => config.readurl = x)
+      opt[String]("writeUrl").action((x, _) => config.writeurl = x)
+      opt[Unit]("countOnly").action((_, _) => config.countonly = true)
```

Comment:
I don't see `countOnly` used in this job. Is that intentional?

Reply:
It's used in the test (https://github.com/mesosphere/spark-build/blob/spark-529-scopt-args-need-equals-sign/tests/test_spark.py#L273) as a boolean flag with no args.

Reply:
OK